### DISCUSSION TITLE
Does Tanstack Table support grid vs list view?

### DISCUSSION THREAD

**Discussion by marceloverdijk** [TanStack/table#4855] [Q&A]
E.g. bootstrap-table has this: 
https://examples.bootstrap-table.com/#extensions/custom-view.html

Like:

<img width="1430" alt="image" src="https://github.com/TanStack/table/assets/197004/d0db4f8f-5d75-4510-9797-53102008f2a1">

<img width="1386" alt="image" src="https://github.com/TanStack/table/assets/197004/258a57c3-2e91-492c-8de6-b87c44cb6af5">

So it's the same "table" and data, but only rendered as a grid and not as traditional <tr><td>.

Both views can be dynamically switched using the top right icon.
This is common pattern when displaying table data vs grid data on web pages.

I wonder if something like this van be implemented?

**Comment by KevinVandy** [comment]
Yeah for sure. You can just make one row and render everything in it. Cells are common, but technically optional

**Reply by marceloverdijk** [reply]
Indeed , that sounds like a good approach. 
But e.g. also the `<thead>` section should be hidden when switching the view type... don't know (yet) how to actually do that.

Do you know if there are some examples in the wild that do something similar?

**Comment by KevinVandy** [comment]
Probably should include here that reading from row.original shows you all data in a single row
